fusor-ember-cli/app/controllers/deployment.js

```diff
@@ -88,12 +88,9 @@ export default Ember.Controller.extend(DeploymentControllerMixin, DisableTabMixi
   ),
 
   hasSubscriptionUUID: Ember.computed(
-    'organizationUpstreamConsumerUUID',
```

Comment:
This property was functionally the same as model.upstream_consumer_uuid and never actually did anything. Think it was left over from an older iteration, so I removed it.